diff --git a/_root.js b/_root.js
index d2852bed4d2e469b3b87ed31df2350047e005ff4..5e2bd0da67b671707c2a746d302f073bbad58839 100644
--- a/_root.js
+++ b/_root.js
@@ -1,9 +1,10 @@
-var freeGlobal = require('./_freeGlobal');
+var freeGlobal = require("./_freeGlobal");
 
 /** Detect free variable `self`. */
-var freeSelf = typeof self == 'object' && self && self.Object === Object && self;
+var freeSelf =
+  typeof self == "object" && self && self.Object === Object && self;
 
 /** Used as a reference to the global object. */
-var root = freeGlobal || freeSelf || Function('return this')();
+var root = freeGlobal || freeSelf || globalThis;
 
 module.exports = root;
